backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/entrytypes/BeaconGranularityTests.java

```diff
@@ -4,9 +4,11 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 
 import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Map;
 import org.junit.jupiter.api.Test;
 import org.molgenis.emx2.beaconv2.QueryEntryType;
 import org.molgenis.emx2.beaconv2.requests.BeaconRequestBody;
+import spark.Request;
```

Comment:
this is not what you should do; don't make your code dependent on spark. Instead pass what you need. It is unnecessary and makes it harder to move to another service framework which we will do coming 6 months.

(this may be a legacy issue; previous beacon impl already included use of spark. Please refactor out if encounter it)

Reply:
This is mostly my fault. I indeed pass spark objects into beacon specific code. Shouldn't have done that. Need to deal with it when we migrate to another framework